src/zk/dist_lock_test.cc

```diff
@@ -43,10 +43,10 @@ void OnLockedCallback() { call_invoked = true; }
 void OnLostCallback() {}
 
 TEST_F(DistLockTest, Lock) {
-    ZkClient client("127.0.0.1:6181", "", 10000, "127.0.0.1:9527", "/rtidb_lock");
+    ZkClient client("127.0.0.1:6181", "", 10000, "127.0.0.1:9527", "/openmldb_lock");
     bool ok = client.Init();
     ASSERT_TRUE(ok);
-    DistLock lock("/rtidb_lock/nameserver_lock", &client, boost::bind(&OnLockedCallback), boost::bind(&OnLostCallback),
+    DistLock lock("/openmldb_lock/nameserver_lock", &client, boost::bind(&OnLockedCallback), boost::bind(&OnLostCallback),
```

Comment:
hi @marc-marcos , please resolve the `cpplint` warning. Lines should be <= 120 characters long